src/paredit/extension.ts

```diff
@@ -12,11 +12,30 @@ let onPareditKeyMapChangedEmitter = new EventEmitter<String>();
 const languages = new Set(["clojure", "lisp", "scheme"]);
 let enabled = true;
 
+
+/**
+ * Copies the text represented by the range from doc to the clipboard.
+ * @param doc
+ * @param range
+ */
+function copyToClipboard(doc: EditableDocument, [start, end]){
```

Comment:
Nice! I suggest a bit clearer name:
```suggestion
function copyRangeToClipboard(doc: EditableDocument, [start, end]){
```

Reply:
Obviously we also need to fix the call sites. 😄